### PR TITLE
Enable the CORS headers / Be able to desable the OGC proxy

### LINKED ISSUE #1226: OGS Proxy - support CORS headers
We should be able to disable the OGS proxy, and we should add the CORS headers in WMS and WMTS:
`Header set Access-Control-Allow-Origin "*"`

External service that enable it:
http://wms.geo.admin.ch
http://wmts.geo.admin.ch

And don't support:
ASIT-VD, Vaud, R-Pod

### PR DESCRIPTION
**Pull request by sbrunner**
Fix #1226

### CODE REVIEW

File: c2cgeoportal/__init__.py [hunk 1]
```diff
@@ -215,11 +215,12 @@ def includeme(config):
     config.add_directive('set_user_validator', set_user_validator)
     config.set_user_validator(default_user_validator)
 
-    # add an OGCProxy view
-    config.add_route(
-        'ogcproxy', '/ogcproxy',
-        custom_predicates=(ogcproxy_route_predicate,))
-    config.add_view('papyrus_ogcproxy.views:ogcproxy', route_name='ogcproxy')
+    if config.get_settings().get("ogcproxy_enable", "True") == "True":
```
Comment (asaunier) [inline c2cgeoportal/__init__.py]: Is this "ogcproxy_enable" parameter advertised somewhere in the doc?

Comment (sbrunner) [inline c2cgeoportal/__init__.py]: Documentation added :-)

File: doc/integrator/security.rst [hunk 2]
```diff
@@ -6,17 +6,34 @@ Security
...
+Default: ``false``
 
 Enable / Disable layer(s) in the WMS GetCapability
 --------------------------------------------------
 
-To hide protected layers from the WMS GetCapabilities, set ``use_security_metadata`` to ``true`` in your ``<package>/config.yaml.in``.
+To hide protected layers from the WMS GetCapabilities, set ``use_security_metadata`` to ``true`` in your ``config.yaml.in``.
 
-Be careful as too many protected layers will cause an error because Apache has a 
+Be careful as too many protected layers will cause an error because Apache has a
```
Comment (asaunier) [inline doc/integrator/security.rst]: Be careful that too many...

**Comment by asaunier** [comment]
Except the small change suggested in the doc, looks good!

**Pull request merged**